### ISSUE TITLE
`valid-test-tags` doesn't work for strings defined with ` (backticks)

### ISSUE THREAD

**Issue by brjakab** [playwright-community/eslint-plugin-playwright#460]
If I have a test like this:
```ts
test(
  `Login and save authentication state`,
  { tag: `@todo` },
  async ({ page }) => {},
);
```

and in my eslint config I extend  `playwright.configs['flat/recommended']`, then the `playwright/valid-test-tags` rule gives an error:
`Tag must be a string or array of strings`

If I supply an array for the tag:
```
 { tag: [`@todo`] },
```

then the rule doesn't signal an error.

But if I change the eslint config to disallow the `@todo` tag
```
"playwright/valid-test-tags": ["error", { "disallowedTags": ["@skip", "@todo"] }]
```
it still doesn't signal an error, so I assume that the `playwright/valid-test-tags` stopped working. It also doesn't signal an error when I write this without the `@`

```ts
 { tag: [`todo`] },
```
